demos/common/cpp/utils/include/utils/common.hpp

```diff
@@ -365,3 +220,23 @@ inline std::string fileNameNoExt(const std::string &filepath) {
     if (pos == std::string::npos) return filepath;
     return filepath.substr(0, pos);
 }
+
+inline void printExecNetworkInfo(const InferenceEngine::ExecutableNetwork& execNetwork, const std::string& modelName,
+    const std::string& deviceName) {
+    slog::info << "Network " << modelName << " is loaded to " << deviceName << " device." << slog::endl;
```

Comment:
We should use common terms in all demos. 
@Wovchena @vladimir-dudnik @fzhar what should we use: `network`, `model`, `net` or other?
P.S.: most py/demos use the `model` term.

Reply:
`model` is ok for me

Reply:
Changed.